core-client/src/main/java/org/glassfish/jersey/client/ClientProperties.java

```diff
@@ -405,6 +405,31 @@
      */
     public static final String REQUEST_ENTITY_PROCESSING = "jersey.config.client.request.entity.processing";
 
+    /**
+     * Registers 100-Continue value for Expect header processing to be set in HttpUrlConnector (default Jersey
```

Comment:
Allows for HTTP Expect:100-Continue being handled by the HttpUrlConnector

Reply:
applied